src/js/middleware.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+// Base grade for express middleware modules...
+//
+// The express module will automatically attempt to wire in anything with this gradeName into itself
+
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.middleware");
+
+gpii.express.middleware.complainAboutMissingFunction = function(){
```

Comment:
I added `"white": true` to the `.jshintrc` and the errors were immediately apparent.  The default in 2.4.0 ([according to the source](https://github.com/jshint/jshint/blob/2.4.0/examples/.jshintrc)) is for that to be `false`.

To ensure that this rule is consistently applied, I would suggest adding that option rather than relying on the previous defaults.